lib/IMAP/IMAPClientFactory.php

```diff
@@ -117,8 +117,9 @@ public function getClient(Account $account, bool $useCache = true): Horde_Imap_C
 				'backend' => $this->hordeCacheFactory->newCache($account),
 			];
 		}
-		if ($this->config->getSystemValue('debug', false)) {
-			$params['debug'] = $this->config->getSystemValue('datadirectory') . '/horde_imap.log';
+		if ($account->getDebug() || $this->config->getSystemValueBool('mail_debug') || $this->config->getSystemValueBool('MAIL_DEBUG')) {
```

Comment:
```suggestion
		if ($account->getDebug() || $this->config->getSystemValueBool('app.mail.debug') || $this->config->getSystemValueBool('MAIL_DEBUG')) {
```

As mentioned before, `mail_` in config.php is used for the server's symfony mailer configuration. We are using `app.mail.` for the [other options](https://github.com/nextcloud/mail/blob/main/doc/admin.md), so we should use it here as well to maintain a bit of consistency. 😉